docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,135 @@
+* Allow flushing cache for specific partition with the
+  [flush_metadata_cache](hive_flush_metadata_cache) system procedure.
+  ({issue}`10385`)
+* Ensure transaction locks in the Hive Metastore are released in case of query
+  failure when querying Hive ACID tables.  ({issue}`10401`)
+* Disallow reading from Iceberg tables when redirects from Hive to Iceberg are
+  not enabled. ({issue}`8693`, {issue}`10441`)
+* Improve performance when reading ORC files with Bloom filters with range
+  predicates. ({issue}`4108`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10646`,
+  {issue}`10512`)
+* Prevent data loss during `DROP SCHEMA` when the schema location contains files
+  deep in the directory tree, but not tables. ({issue}`10485`)
+* Fix inserting into transactional table when `task_writer_count` > 1.
+  ({issue}`9149`)
+
+## Iceberg connector
+
+* Improve select query planning performance after write operations from Trino.
+  ({issue}`9340`)
+* Add Iceberg `$properties` system table.  ({issue}`10480`)
+* Add support for `ALTER TABLE .. EXECUTE OPTIMIZE` statement. ({issue}`10497`)
+* Respect Iceberg column metrics mode when writing. ({issue}`9938`)
+* Improve performance when reading ORC files with Bloom filters with range
+  predicates. ({issue}`4108`)
+* Ensure table statistics are accumulated in a deterministic way from Iceberg
+  column metrics. ({issue}`9716`)
+* Prevent data loss during `DROP SCHEMA` when the schema location contains files
+  deep in the directory tree, but not tables. ({issue}`10485`)
+* Fix materialized view refresh when view a query references the same table
+  multiple times. ({issue}`10570`)
+
+## MySQL connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Oracle connector
+
+* Map Oracle `date` to Trino `timestamp(0)` type. ({issue}`10626`)
+* Fix performance regression of predicate pushdown on indexed `date` columns.
+  ({issue}`10626`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Phoenix connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Pinot connector
+
+* Add support for basic authentication. ({issue}`9531`)
+
+## PostgreSQL connector
+
+* Add support for [renaming schemas](/sql/alter-schema). ({issue}`8939`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Redshift connector
+
+* Add support for [renaming schemas](/sql/alter-schema). ({issue}`8939`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## SingleStore (MemSQL) connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## SQL Server connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## SPI
+
+* Remove legacy table layout objects. ({issue}`781`)
```

Comment:
In particular, we've removed support for the `getTableLayout` SPI. We should mention that.